src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -23,11 +23,27 @@ public abstract partial class DeriveBytes : System.IDisposable
         public abstract byte[] GetBytes(int cb);
         public abstract void Reset();
     }
+    public abstract partial class ECCurve
+    {
+        protected ECCurve() { }
+    }
+    public enum ECCurveType : int
+    {
+        PrimeMontgomery = 3,
+        PrimeShortWeierstrass = 1,
+        PrimeTwistedEdwards = 2,
+        Unspecified = 0,
+    }
```

Comment:
This is the 'ref' file... I need to re-gen that. I added the new value: Characteristic2 = 4